### ISSUE TITLE
Some improvements for the PDF of the manual

### ISSUE THREAD

**Issue by ViralBShah** [JuliaLang/julia#35495]
- [x] Remove release notes from the PDF (we start with 25 pages of release notes - mostly blank) (https://github.com/JuliaLang/julia/pull/36312)
- [x] The spacing for chapter numbers in the TOC is not sufficient (https://github.com/JuliaDocs/Documenter.jl/pull/1785)

Nice to have: 
- [ ] Have a mechanism for a custom cover page (https://github.com/JuliaDocs/Documenter.jl/pull/1788)

Edit: I updated the title of this issue to cover various issues in the PDF of the manual.

Basically have to go 50 pages to get to the "Getting started" part of the manual, which is chapter 14.

**Comment by ViralBShah** [comment]
I suspect this happens because in the web version, the next page from the landing page is the release notes. However, in the online docs, one probably uses the toc on the left for navigation.

**Comment by mortenpi** [comment]
The reason is that the [`NEWS.md` page, while "hidden", is actually the same level as the Home/Manual/Base/etc. chapters](https://github.com/JuliaLang/julia/blob/cc6e121386758dff6ba7911770e48dfd59520199/doc/make.jl#L53), and so it becomes a separate `\chapter`. It might be possible to make it a lower-level page in the PDF or something along those lines.

**Comment by ViralBShah** [comment]
I guess showing release notes to someone that early on in even in the web version doesn't seem like a good idea. Perhaps we can include it as a link rather than as a separate page? The level of information in the NEWS file as we currently ship it is significantly more technical than the rest of the manual.

**Comment by ViralBShah** [comment]
Actually, is it possible to remove the NEWS altogether from the PDF? It is good the way it is in the web version, but doesn't add much value in the pdf.

**Comment by ViralBShah** [comment]
The TOC has way too much detail. Would be nice to present fewer levels of information in the Table of Contents.

**Comment by ViralBShah** [comment]
Spacing issues in TOC:  

![Screen Shot 2020-04-27 at 6 14 57 PM](https://user-images.githubusercontent.com/744411/80426085-043f4400-88b3-11ea-9b7a-b9cbf5add40b.png)

**Comment by ViralBShah** [comment]
How does one go about fixing the latex generated that finally builds this PDF?

**Comment by ViralBShah** [comment]
Self-answer: https://github.com/JuliaDocs/Documenter.jl/blob/master/src/Writers/LaTeXWriter.jl

**Comment by inkydragon** [comment]
>  Present lesser levels of subchapters in the TOC (It is almost 20 pages right now)

How about only show "Chapter" title + compact spacing?

[TOC-baseline][baseline] (14p) v.s. [compact-spacing][show-chapter] (3p)

patch: [en-toc-patch][en-toc-patch]

![image](https://user-images.githubusercontent.com/5158738/148416286-25c71fcf-4b13-4df9-9f5b-691e3b46c276.png)


[baseline]: https://drive.google.com/file/d/1MwbPlsfMvC2xoK1mWv6XehPLat87IMbw/view?usp=sharing
[show-chapter]: https://drive.google.com/file/d/1XzINy5xvf6UQHYXYD3dSlfYIKN5eQjUZ/view?usp=sharing
[en-toc-patch]: https://github.com/inkydragon/Julia-LaTeX-doc/compare/9db176eae3d1587964707b3fa6e4044ac63b8157...en-toc

**Comment by ViralBShah** [comment]
Yes let's do it.

**Comment by inkydragon** [comment]
> Have a cover page with some design and Julia logo

I've designed a cover page based on julia's homepage.
And there are many other choices: [titlepages.pdf](http://tug.ctan.org/info/latex-samples/TitlePages/titlepages.pdf).

![image](https://user-images.githubusercontent.com/5158738/148499996-61ac9447-06dc-4d05-ad15-85da9eeeca45.png)

patch:
+ [custom.sty](https://github.com/inkydragon/Julia-LaTeX-doc/blob/en-cover-page/doc-en/custom.sty)
+ [title.tex](https://github.com/inkydragon/Julia-LaTeX-doc/blob/en-cover-page/doc-en/titlepage/title.tex)
+ [julia-doc-en.tex](https://github.com/inkydragon/Julia-LaTeX-doc/blob/en-cover-page/doc-en/julia-doc-en.tex#L29-L30)

## play with it

The tex code in my repo is self-contained, you don't need to clone julia's repo to compile PDF documents.
You just need to select the right branch ([`en-cover-page`][en-cover-page]), clone the repo, go into the right folder (`doc-en/`), and [build][] the PDF.

[en-cover-page]: https://github.com/inkydragon/Julia-LaTeX-doc/tree/en-cover-page
[build]: https://github.com/inkydragon/Julia-LaTeX-doc/tree/en-cover-page#build

**Comment by ViralBShah** [comment]
@inkydragon May I request you to submit a PR? This would all be great to have.

**Comment by inkydragon** [comment]
Of course, I had the pleasure of designing the cover for julia's document.

My plan is: After #43698 is merged *or* I get some feedback, I'll open a new pr.

The main problem: finding a suitable way to customize the Documenter style.
It's fine to put everything into `make.jl`, but there must be a cleaner way to do it.

**Comment by ViralBShah** [comment]
Also cc @fredrikekre

**Comment by inkydragon** [comment]
**Good News and Bad News**
The good news is that we are about to fix all the problems mentioned in this issue.
The bad news is: I found more issues.

> Take [julia-1.8.0-DEV.pdf](https://raw.githubusercontent.com/JuliaLang/docs.julialang.org/assets/julia-1.8.0-DEV.pdf) as an example.

New todo list:

**Serious issues affecting reading**
+ Internal Links #43652.  
    Almost all internal links are dead except for the link pointing to base/stdlib
+ p453 (Chap 39 Unicode Input): Pick the right font to display the Unicode Input table.  
    You may need both math font and emoji font.
+ p456 (40.2 Command-line switches for Julia): Long table truncation
+ p190 (15.4 Strided Arrays): Long form item line break
+ p1416 (before "Build dependencies"): md link conversion failed 
+ p1418 (104.2 Linux/Architecture Customization): HTML conversion failed.  
    HTML and markdown are mixed in the table.


**Optional improvements**
+ Centering the tables
+ Do not show empty entries in "Release Notes"
+ ...

**Comment by ViralBShah** [comment]
I think we can remove the release notes from the manual altogether, and instead just link to them.

**Comment by ViralBShah** [comment]
With the improvements discussed in https://github.com/JuliaLang/julia/pull/43698#issuecomment-1089051922, all the relevant items have been taken care of. I feel comfortable closing this.

A big shoutout to @inkydragon for making most of this happen.

**Comment by ViralBShah** [comment]
The only nice to have thing is to see if the Julia logo can be included on the cover page. We may already have it checked into the julia repo.

**Comment by ViralBShah** [comment]
Thanks @inkydragon for seeing this through across multiple PRs!
